packages/abi-coder/src/types/TEncodingOptions.ts

```diff
@@ -56,55 +49,14 @@ type Primitive = string | number | boolean;
  *    • configurable
  *
  *
- * Version 1:
+ * Encoding Version 1:
  *
  * It currently is supported only by logs, but has no specific configurable options.
  * More information on the improvements made in this version can be found in the
  * fuel specs (https://github.com/FuelLabs/fuel-specs/blob/master/src/abi/argument-encoding.md#version-1
  */
-export type EncodingOptions = {
-  version?: number;
+export type TEncodingOptions = {
```

Comment:
IMO there is no need for the `T` prefix as it doesn't add any additional description to the concept the type is representing.